src/SonarScanner.MSBuild.PreProcessor/SonarWebService.cs

```diff
@@ -270,7 +270,7 @@ public Task<AnalysisCacheMsg> DownloadCache(string projectKey, string branch)
 
             logger.LogDebug(Resources.MSG_DownloadingCache, projectKey, branch);
             var uri = GetUri("/api/analysis_cache/get?project={0}&branch={1}", projectKey, branch);
-            return downloader.DownloadStream(uri).ContinueWith(x => AnalysisCacheMsg.Parser.ParseFrom(x.Result));
+            return downloader.DownloadStream(uri).ContinueWith(x => x.Result is null ? null : AnalysisCacheMsg.Parser.ParseFrom(x.Result)); // FIXME: Remove workaround, add UTs
```

Comment:
You can rebase on https://github.com/SonarSource/sonar-scanner-msbuild/pull/1415